include/mc_tasks/lipm_stabilizer/StabilizerTask.h

```diff
@@ -743,6 +843,25 @@ struct MC_TASKS_DLLAPI StabilizerTask : public MetaTask
   /**< Whether the estimator needs to be reset (robot in the air, initialization) */
   bool dcmEstimatorNeedsReset_ = true;
 
+  /** Adding an offset to the CoM for the predictable / measurable external wrenches on the robot surface. */
```

Comment:
If you want to group together the entries related to the external wrenches in the doxygen, you can use
```cpp
/** @name Members related to stabilization in the presence of external wrenches
 * 
 * You may refer to [RA-L/IROS 2021] for further details
 * @{
 */
[...members...]
/** @} */
```

Reply:
Updated in https://github.com/mmurooka/mc_rtc/commit/4af4cff4e51ea4d7f225816f2c07be86e6c36e5d. As for the paper, I'd like to add it, but it's still under review... I'll add it when it is successfully published.